### PR TITLE
SUM-113 - addendum: adding pill background variant

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY FOR REVIEW

# Summary
-  Adding pill card variant: https://stanfordits.atlassian.net/browse/SUM-39?focusedCommentId=151207&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-151207

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by mdyoung3** [comment]
@rebeccahongsf Adding you to make sure about the colors here.

File: docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php [hunk 1]
```diff
@@ -38,6 +38,19 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       ],
       '#default_value' => $paragraph->getBehaviorSetting('su_card_styles', 'sum_card_variant'),
     ];
+    $element['sum_card_bg_color_variant'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Banner Background Color'),
```
Comment (pookmish) [inline docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php]: I think you mean "Card"

**Review by pookmish — CHANGES_REQUESTED** [review]

File: docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php [hunk 2]
```diff
@@ -38,6 +38,19 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
...
+    $element['sum_card_bg_color_variant'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Card Background Color'),
+      '#empty_option' => $this->t('Poppy Light'),
+      '#options' => [
+        'semitransparent_poppy' => $this->t('Semitransparent Poppy Light'),
+        'olive' => $this->t('Olive Light'),
+        'semitransparent_olive' => $this->t('Semitransparent Olive Light'),
+        'spirited' => $this->t('Spirited Light'),
+        'semitransparent_spirited' => $this->t('Semitransparent Spirited Light'),
```
Comment (rebeccahongsf) [inline docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:50]: Colors look good to me, @mdyoung3 😃 Thank you!

File: docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php [hunk 3]
```diff
@@ -38,6 +38,19 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       ],
       '#default_value' => $paragraph->getBehaviorSetting('su_card_styles', 'sum_card_variant'),
     ];
+    $element['sum_card_bg_color_variant'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Card Background Color'),
```
Comment (rebeccahongsf) [inline docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php]:
```suggestion
      '#title' => $this->t('Pill Card Background Color'),
```
@mdyoung3 should we call this 'Pill Card Background Color' instead? So content authors know it's for the Pill Card only 😃  Unless we can conditional render this select field to only display when the user selects Pill from the Card Style.

**Review by pookmish — APPROVED** [review]

File: docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php [hunk 4]
```diff
@@ -38,6 +38,19 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
...
+      '#empty_option' => $this->t('Poppy Light'),
+      '#options' => [
+        'semitransparent_poppy' => $this->t('Semitransparent Poppy Light'),
+        'olive' => $this->t('Olive Light'),
+        'semitransparent_olive' => $this->t('Semitransparent Olive Light'),
+        'spirited' => $this->t('Spirited Light'),
+        'semitransparent_spirited' => $this->t('Semitransparent Spirited Light'),
+      ],
+      '#default_value' => $paragraph->getBehaviorSetting('su_card_styles', 'sum_card_bg_color_variant'),
+    ];
```
Comment (pookmish) [inline docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:53]: Optionally, you could add `#states` to show the field only when a pill is selected.

**Pull request merged**